client/components/ScenariosList/ScenarioCard.jsx

```diff
@@ -1,193 +1,160 @@
 
-    this.onRestoreClick = this.onRestoreClick.bind(this);
-  }
+  const cardHeaderAriaLabel = isMultiParticipantScenario
+    ? `${title} is a multi-participant scenario.`
+    : `${title} is a solo scenario.`;
 
-  async onRestoreClick(event, { name }) {
+  const onRestoreClick = async (event, { name }) => {
     if (name === 'restore') {
-      const originalScenario = this.state.scenario;
-      originalScenario.deleted_at = null;
+      scenario.deleted_at = null;
 
-      // TODO: move to async action
-      await (await fetch(`/api/scenarios/${originalScenario.id}`, {
-        method: 'POST',
+      await (await fetch(`/api/scenarios/${scenario.id}`, {
+        method: 'PUT',
         headers: {
           'Content-Type': 'application/json'
         },
-        body: JSON.stringify(originalScenario)
+        body: JSON.stringify(scenario)
       })).json();
 
-      // Revive scenario by requesting restored record from server
-      await this.props.getScenario(originalScenario.id);
+      dispatch(getScenario(scenario.id));
     }
-  }
-
-  render() {
-    const { onRestoreClick } = this;
-    const { onClick, showActions = true, user } = this.props;
-    const { scenario } = this.state;
-    const { categories = [], id, description, deleted_at, title } = scenario;
-    // eslint-disable-next-line no-unused-vars
-    const officialCheckmark = categories.includes('official') ? (
-      <Icon name="check" aria-label="Official" className="primary" />
-    ) : null;
-
-    const isAuthorized =
-      scenario.author_id === user.id || user.roles.includes('super_admin');
-
-    const ariaLabelledby = Identity.id();
-    const ariaDescribedby = Identity.id();
-
-    const updatedAgo = Moment(scenario.updated_at).fromNow();
-    const createdAgo = Moment(scenario.created_at).fromNow();
-
-    const scenarioUpdatedOrCreatedTime = scenario.updated_at ? (
-      <p>
-        Last edited{' '}
-        <time className="sc__time" dateTime={scenario.updated_at}>
-          {updatedAgo}
-        </time>
-      </p>
-    ) : (
-      <p>
-        Created{' '}
-        <time className="sc__time" dateTime={scenario.created_at}>
-          {createdAgo}
-        </time>
-      </p>
-    );
-
-    const statusItems = {
-      1: { type: 'Draft', icon: 'edit' },
-      2: { type: 'Public', icon: 'eye' },
-      3: { type: 'Private', icon: 'eye slash' }
-    };
-    const status = statusItems[scenario.status];
-
-    const clickables = {
-      onClick,
-      onKeyUp(...args) {
-        Events.onKeyUp(...args, onClick);
-      }
-    };
-
-    const isMultiParticipantScenario = scenario.personas.length > 1;
-
-    const cardHeaderIconClassName = isMultiParticipantScenario
-      ? 'users'
-      : 'user';
-
-    const cardHeaderAriaLabel = isMultiParticipantScenario
-      ? `${title} is a multi-participant scenario.`
-      : `${title} is a solo scenario.`;
-
-    return deleted_at ? (
-      <Gate isAuthorized={isAuthorized}>
-        <DeletedCard
+  };
```

Comment:
This refactoring opportunity would be a great time to move this into `client/actions/scenario.js`